packages/sources/token-balance/docs/known-issues.md

```diff
@@ -0,0 +1,9 @@
+## Usage Notes
```

Comment:
Not sure if this file should be named known-issues

Reply:
Also don't we already have this in readme?

Reply:
right now it's the only way to get in additional README notes without adding to the blacklist.

Can update the scripts after delivery, but for now it's either this or hardcode in the README + blacklist